.travis.yml

```diff
@@ -10,13 +10,20 @@ before_install:
   - sudo apt-get install -qq g++-9
   - export CXX="g++-9" CC="gcc-9"
   - sudo apt-get install git
+  - sudo apt-get install ninja-build python3-pip python3-setuptools
```

Comment:
Why don't you use the addon: apt ?

Reply:
Don't know. Is one way better than the other?

Reply:
I guess the only benefit would be that you wouldn't have to worry about the individual apt commands and they would only run on linux builds (in case macOS builds are added later) https://docs.travis-ci.com/user/installing-dependencies/

Although if it's prefered to see the individual commands, this would be fine